src/app/components/SearchResultsPage/SearchResultsPage.jsx

```diff
@@ -143,7 +135,7 @@ const SearchResultsPage = (props, context) => {
                     total={totalHits}
                     perPage={50}
                     page={parseInt(page, 10)}
-                    to={{ pathname: urlSearchString }}
```

Comment:
If we don't pass `to` as a props here, `this.props.to` will be ` to: { pathname: '#' },` in `<Pagination>` and I think it will break `<Link>`?